index.js

```diff
@@ -0,0 +1,269 @@
+'use strict';
+let _ = require('lodash')
+  , redis = require('redis')
+  , modelQueue = false
+  , schemas = {};
+
+class ModelQueue {
```

Comment:
Same comment about the global `ModelQueue` instance vs. attaching a single `ModelQueue` to each `Eriksen` instance...?